cookiecutter/generate.py

```diff
@@ -284,8 +287,15 @@ def generate_files(repo_dir, context=None, output_dir='.',
 
     _run_hook_from_repo_dir(repo_dir, 'pre_gen_project', project_dir, context)
 
+    # Configure jinja templates searchpath
+    templates = ['.']
+    if extra_templates is not None:
+        if isinstance(extra_templates, basestring):
```

Comment:
you are right, I either 
* improve the documentation (to make clear that a single extra_template is accepted as an string) or
* allow only lists.

I prefer the first option